src/plugins/controls/common/options_list/types.ts

```diff
@@ -17,6 +17,7 @@ export interface OptionsListEmbeddableInput extends DataControlInput {
   selectedOptions?: string[];
   runPastTimeout?: boolean;
   singleSelect?: boolean;
+  exclude?: boolean;
```

Comment:
`exclude` should be treated the same as `selectedOptions` - i.e.  toggling its value should trigger unsaved changes, the author should be able to save in a default value for a given control, etc. Hence, why it's part of `explicitInput` and not just component state.